src/libraries/NetCoreAppLibrary.props

```diff
@@ -48,7 +48,6 @@
       System.IO.FileSystem.Watcher;
       System.IO.IsolatedStorage;
       System.IO.MemoryMappedFiles;
-      System.IO.Pipelines;
```

Comment:
Is removing pipelines reference here going to require a reaction in aspnetcore? Did we make changes to not reference it ourselves? @halter73 @Tratcher @JunTaoLuo